### DISCUSSION TITLE
How to change store of admin menu list from redis to sql db,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#16498] [Q&A]
How to change store of admin menu list from redis to sql db,please?

**Comment by Piedone** [comment]
If you mean the user-editable menu by the [Admin Menu module](https://docs.orchardcore.net/en/latest/reference/modules/AdminMenu/), then that's stored in the DB already. It's only cached in Redis (but only if you enable Redis, otherwise in local memory) via `IDocumentManager`. If you want to change that, you'll need https://github.com/OrchardCMS/OrchardCore/issues/16149.

**Comment by effapp** [comment]
Hi @Piedone ,
     Yes, I mean the user-editable menu by the admin menu module. The admin menu list still exists in redis when I deleted the entire site.
The admin menu list that cached in redis always appears in new tenant site.

I mannually delete admin menu key of redis after deleting entire site and the issue is resolved. 

Thanks.

**Reply by Piedone** [reply]
OK then!
